src/app/js/fields/wizard/TabGeneral.js

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import { Field, FieldArray, formValueSelector } from 'redux-form';
+import translate from 'redux-polyglot/dist/translate';
+import { compose } from 'recompose';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { TextField as MUITextField, Box } from '@mui/material';
+
+import FieldLabelInput from '../FieldLabelInput';
+import FieldInternal from '../FieldInternal';
+import SourceValueToggleConnected from '../sourceValue/SourceValueToggle';
+import TransformerList from '../transformers/TransformerList';
+
+import {
+    field as fieldPropTypes,
+    polyglot as polyglotPropTypes,
+} from '../../propTypes';
+import { FIELD_FORM_NAME } from '..';
+
+export const isSubresourceTransformation = transformers => {
+    const operations = transformers.map(t => t.operation).join('|');
+    return (
+        operations === 'COLUMN|PARSE|GET|STRING|REPLACE_REGEX|MD5|REPLACE_REGEX'
+    );
+};
+
+export const renderTransformer = (
```

Comment:
A function that return a function.
Could be directly a component

Reply:
it's because redux form Field does not accept JSX component with props